src/Classes/Options/Particles/RotateAnimation.ts

```diff
@@ -0,0 +1,30 @@
+import {IRotateAnimation} from "../../../Interfaces/Options/Particles/IRotateAnimation";
+import {Utils} from "../../Utils/Utils";
+
+export class RotateAnimation implements IRotateAnimation {
+    public enable: boolean;
+    public speed: number;
+    public sync: boolean;
+
+    constructor() {
+        this.enable = false;
+        this.speed = 0;
+        this.sync = false;
+    }
+
+    public load(data: IRotateAnimation): void {
+        if (Utils.hasData(data)) {
+            if (Utils.hasData(data.enable)) {
+                this.enable = data.enable;
+            }
+
+            if (Utils.hasData(data.speed)) {
+                this.speed = data.speed;
+            }
+
+            if (Utils.hasData(data.sync)) {
+                this.sync = data.sync;
+            }
+        }
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5164352)